<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addContractWin</class>
 <widget class="QWidget" name="addContractWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Contract</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>235</width>
     <height>234</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="descBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>113</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Notify Before:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QSpinBox" name="notifyMonthsVal"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Months</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QSpinBox" name="notifyDaysVal"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Days</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Expiry:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="expiryDate">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
